CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning].
 
 This release introduces complete floating-point support.
 
-With that in place, Fizzy passes all(*) of the [official test suite (spectest 1.0)]:
+With that in place, Fizzy passes all(*) of the [official test suite (spectest 1.0)](https://github.com/wasmx/wasm-spec/releases/tag/w3c-1.0-jsontests-20200813):
```

Comment:
Should these link to `releases/tag/w3c-...` or `tree/w3c-..` ?

Reply:
Maybe `https://github.com/wasmx/wasm-spec/tree/w3c-1.0-jsontests-20200813/test/core/json`?